docs/tutorial/02-draw-a-snake.md

```diff
@@ -206,28 +220,26 @@ graph = draw_object(@graph, snake)
 
 To keep things organized, we'll implement our own `draw_object/2` helper function which will turn our abstract snake object into actual drawable objects for Scenic. The function takes two arguments: the graph and our snake object.
 
-> Coach: Talk about pipelines.
-
-For each square of the snake's body, we draw a tile one our grid.
+For each square of the snake's body, we draw a tile on our grid.
```

Comment:
```suggestion
For each square of the snake's body, we fill a cell on our grid.
```

Let's try to stick to one word for cell / tile if possible. I personally think that "cell" is a better fit and likely to be understood easier by more people.